documentation/latest/operations/requirements.md

```diff
@@ -20,21 +20,21 @@ needed for the device-agent.
 
 | Requirement       | Min version | Optional  |
 | yggdrasil         | master      | no        |
-| podman            | >=4         | no        |
+| podman            | >=4.1       | no        |
```

Comment:
should we update the code here as well?
https://github.com/project-flotta/flotta-device-worker/blob/278d2585a05d010b5353b6ac5482bf68c5cc3c0b/internal/workload/podman/podman.go#L152